webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -15113,6 +15093,38 @@ var userOverride = {
                     $file_progress.removeClass('n-vis').addClass('vis');
                 }
             };
+            _this.handleEncryptedElements = function(encryptedElements){
+                for(var i = 0; i < encryptedElements.length; i++){
+                    switch (encryptedElements[i].tagName) {
+                        case 'IMG':
+                            var img = encryptedElements[i];
+                            if (KommunicateUI.isInView(img, document.querySelector('#mck-message-cell .mck-message-inner'))) {
```

Comment:
isInView code looks to be same in all conditions, if that's the case, then it should be outside of switch statement. and inside code should have only what is unique to that part

Reply:
moved it out